/home/leonhard/anaconda3/envs/tactile_envs/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/home/leonhard/anaconda3/envs/tactile_envs/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
Traceback (most recent call last):
  File "/home/leonhard/workshop/test/M3L/trainDINO.py", line 264, in <module>
    main()
  File "/home/leonhard/workshop/test/M3L/trainDINO.py", line 231, in main
    model.learn(
  File "/home/leonhard/workshop/test/M3L/models/ppo_dino.py", line 451, in learn
    return super().learn(
           ^^^^^^^^^^^^^^
  File "/home/leonhard/anaconda3/envs/tactile_envs/lib/python3.11/site-packages/stable_baselines3/common/on_policy_algorithm.py", line 313, in learn
    self.train()
  File "/home/leonhard/workshop/test/M3L/models/ppo_dino.py", line 420, in train
    self.dino.on_train_epoch_end()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leonhard/anaconda3/envs/tactile_envs/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1695, in __getattr__
    raise AttributeError(f"'{type(self).__name__}' object has no attribute '{name}'")
AttributeError: 'DinoVisionTransformer' object has no attribute 'on_train_epoch_end'